src/common/__tests__/merjoonTransformer.test.ts

```diff
@@ -148,5 +157,33 @@ describe('MerjoonTransformer', () => {
         });
       });
     });
+    
+    describe('HTML_TO_STRING', () => {
+      it('should return string parsed from html', () => {
+        const data = { description: '<ol>' +
```

Comment:
this is very simple one,

let's first create test data using everything from here
![image](https://github.com/user-attachments/assets/cdc15dd7-7918-4e19-aefd-33f41c805472)
then use it to write proper tests with several html elements. in your tests please remove all the `\n`s from the html, make sure that `\n`s are added properly despite the `\n`s in the html